python/GangaLHCb/Lib/Applications/BenderBox.py

```diff
@@ -0,0 +1,520 @@
+# -*- coding: utf-8 -*-
```

Comment:
@alexanderrichards  for sure it is not nesessary. But if I write my name in comments using my language, this line will be very usefull....  anyhow it does not harm..